Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -101,6 +107,41 @@ public AssemblyDefinition VerifyAndGetAssembly()
         {
             GlobalType = Module.GetType("<Module>");
         }
+        _globalTypeStaticCtor = new(
+            () =>
+            {
+                var ctor = BuildStaticCtor();
+                GlobalType.Methods.Add(ctor);
+                return ctor;
+            });
+    }
+
+    private MethodDefinition BuildStaticCtor()
+    {
+        var methodAttributes = MethodAttributes.Private
+                               | MethodAttributes.HideBySig
+                               | MethodAttributes.SpecialName
+                               | MethodAttributes.RTSpecialName
+                               | MethodAttributes.Static;
+        MethodDefinition method = new MethodDefinition(".cctor", methodAttributes, Module.TypeSystem.Void);
+        method.Body.Instructions.Add(Instruction.Create(OpCodes.Ret));
+        return method;
+    }
+
+    internal void AddFieldInitialization(FieldDefinition field, Ir.Expressions.IExpression value)
+    {
+        var lval = new Ir.Expressions.LValues.LValueGlobalVariable(field);
+        var context = new TranslationUnitContext(this);
+        var scope = new FunctionScope(context, GlobalTypeStaticCtor);
+        var lastOpcode = GlobalTypeStaticCtor.Body.Instructions.Last();
+        if (lastOpcode.OpCode == OpCodes.Ret)
+        {
+            GlobalTypeStaticCtor.Body.Instructions.Remove(lastOpcode);
+            lval.EmitSetValue(scope, value);
+            GlobalTypeStaticCtor.Body.Instructions.Add(lastOpcode);
+        }
+        else
```

Comment:
This `else` statement could be omitted calling `return` as last instruction inside `if`.

Also I recommend to watch the video about if-else refactors: https://youtu.be/_ougvb8mT7k

Reply:
Why?